### PR TITLE
feat: move policy code to publish

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** @seve

**Readability:** @tihuan  

---

## Changes
Fixing test from https://github.com/chanzuckerberg/single-cell-data-portal/pull/1575

### CODE REVIEW

File: tests/functional/backend/corpora/test_api.py [hunk 1]
```diff
@@ -49,7 +49,6 @@ def test_collection_flow(self):
         data = {
             "contact_email": "lisbon@gmail.com",
             "contact_name": "Madrid Sparkle",
-            "data_submission_policy_version": "1",
```
Comment (Bento007) [inline tests/functional/backend/corpora/test_api.py:52]: changed to fix functional tests

File: tests/functional/backend/corpora/test_api.py [hunk 2]
```diff
@@ -124,7 +126,6 @@ def test_delete_private_collection(self):
...
```
Comment (Bento007) [inline tests/functional/backend/corpora/test_api.py:127]: changed to fix functional tests

File: tests/functional/backend/corpora/test_api.py [hunk 3]
```diff
@@ -164,7 +165,6 @@ def test_dataset_upload_flow(self):
         body = {
             "contact_email": "lisbon@gmail.com",
             "contact_name": "Madrid Sparkle",
-            "data_submission_policy_version": "1",
```
Comment (Bento007) [inline tests/functional/backend/corpora/test_api.py:167]: changed to fix functional tests

File: tests/functional/backend/corpora/test_revisions.py [hunk 4]
```diff
@@ -20,7 +20,6 @@ def create_collection(self, headers):
         data = {
             "contact_email": "lisbon@gmail.com",
             "contact_name": "Madrid Sparkle",
-            "data_submission_policy_version": "1",
```
Comment (Bento007) [inline tests/functional/backend/corpora/test_revisions.py:23]: changed to fix functional tests

**Review by Bento007 — COMMENTED** [review]
These are the new changes

**Review by tihuan — APPROVED** [review]

**Review by seve — APPROVED** [review]

**Comment by seve** [comment]
Before merge can we have a better commit name please

**Pull request merged**